readthedocs/proxito/tests/test_old_redirects.py

```diff
@@ -1135,3 +1144,34 @@ def setUp(self):
             default_true=True,
             future_default_true=True,
         )
+
+    def test_redirect_prefix_crossdomain_with_newline_chars(self):
+        """
+        Overridden to make it compatible with the new implementation.
```

Comment:
We could also make the old implementation compatible, isn't a lot of code, but I'm not sure if we want to spend more time there... (but kind of already spent time debugging it :_)

Reply:
"The old implementation", is the one we are going to delete, right?

Reply:
yeah